packages/api-media-karma/__tests__/specs/index.ts

```diff
@@ -162,11 +172,12 @@ describe('Karma Media Adapter', () => {
     })
 
     test('GetImageURL should return a string with transformation', async () => {
-      const testImageWithFocal: Image = {
+      const testImageWithFocal: ImageWithFocalPoint = {
```

Comment:
As SQL is mostly flat and FocalPoint was an object, it's now in a different table.
Image itself does not contain the focal point anymore, but it has to be added via `includes: {focalPoint: true}`, this will create a type `Image & {focalPoint: FocalPoint}` by Prisma, to reuse it and not having to set that everytime, I added `ImageWithFocalPoint`